crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -578,8 +578,9 @@ end
 #!
 #! Where:
 #! - version is the protocol version of the transaction reference block.
+const.BLOCK_METADATA_VERSION_PTR=BLOCK_METADATA_PTR+1
 export.get_blk_version
-    padw mem_loadw.BLOCK_METADATA_PTR drop drop swap drop
+    mem_load.BLOCK_METADATA_VERSION_PTR
 end
```

Comment:
I'm not sure about this pattern of defining a constant directly above the function. Does the `#!` doc comments apply to the constant now or still to the procedure? And for a better general overview, I think it makes more sense to define them where all other constants are, e.g. here next to `BLOCK_METADATA_PTR`.

Reply:
> And when working with this code, it seems easier to have these related constants next to each other than scattered to see the relationship directly. A small point is also that reviewing changes to this is easier if these related constants are locally grouped.

That's a very good point 👍🏼